src/content/rooms/medium/library.coffee

```diff
@@ -43,7 +43,7 @@ Job.Library::next = add class LibraryDaily extends Page
     if Math.random() < 0.75  or g.events.LibraryDaily?[0] is g.day then return false
     l = [
       """|| bg="Library/Belly.jpg"
-        -- "Can I help you find anything, #{if @worker.gender is 'f' then "ma'am" else 'sir'}? M-my shirt? Yes, I-I suppose, if you wish..." """,
+        -- "Can I help you find anything, #{sir}? M-my shirt? Yes, I-I suppose, if you wish..." """,
```

Comment:
This needs to be "#{sir @worker}" - the page hasn't specified who to use yet, so you'll get indeterminate results (the last person referenced on a previous page, or an error)